src/arguments.rs

```diff
@@ -28,21 +31,22 @@ pub enum Command {
         about = "Start a new time entry. Call with no arguments to start in interactive mode."
     )]
     Start {
+        #[structopt(short, long)]
+        interactive: bool,
+        #[structopt(help = "Description of the time entry.")]
+        description: Option<String>,
         #[structopt(
-            help = "Description of the time entry. If not specified, the user will be prompted to enter it."
+            short,
+            long,
+            help = "Exact name of the project you want the time entry to be associated with."
         )]
-        description: Option<String>,
-        #[structopt(short, long)]
         project: Option<String>,
         #[structopt(short, long)]
-        billable: bool,
+        billable: Option<bool>,
```

Comment:
This change prevents you from passing billable as a boolean flag, previously this would work

```console
$ toggl start --description My Work --billable
```

Also the billable flag doesn't work as expected you are always prompted to configure it in `--interactive` mode

https://user-images.githubusercontent.com/1716853/233838570-2ed4a69e-82a9-47a1-8258-ce2db9f64219.mov

Reply:
I'd suggest changing this back to just a `bool`

If the user doesn't pass billable we can still check the project default to decide on it.
If `--billable` is passed we adhere to that.

Reply:
The problem is: How do we know if billable was passed or not if it's not an Option<bool>? 🤔

Reply:
As in: An user can opt-in, but he can't opt-out of billable. Maybe that's fine and the ergonomics of `--billable` are indeed better

Reply:
Boolean flags default to `false` so the user can only ever pass a boolean true flag.
There is no  valid state where the user is passing a false flag, to do so you just omit the flag.

```console
$ toggl start --billable # Start a billable time entry regardless of other settings


$ toggl start --project watercooler-labs # Start time entry use the `billable` default from the project configuration


$ toggl start --project watercooler-labs --billable # Start time entry set `billable` true regardless of project configuration
```

Reply:
What I mean is that you can't pass `false` regardless of project configuration. It's a state we can't represent. Maybe that's fine because it doesn't matter and the convenience of just passing billable as a flag is better

Reply:
Yeah I agree it's a tricky state

On the webapp when you select a project with billable default the time-entry is automatically marked as such.
So to mark it as non-billable you must manually toggle the option.

I think the cli can stick to the simpler states, unless we switch `billable` to a more explicit string flag
with valid options being `""`, `"true"` and `"false"`

```
--billable # no argument defaults to true
--billable true # alias for --billable
--billable false # explicit override
```

Reply:
Yeah, let's keep it simple for now and if people complain we can always change it